docs/source/reference/kubernetes/index.rst

```diff
@@ -0,0 +1,154 @@
+         Kubernetes   2CPU--2GB         2       2         -              kubernetes    0.00          ✔
+         AWS          m6i.large         2       8         -              us-east-1     0.10
+         Azure        Standard_D2s_v5   2       8         -              eastus        0.10
+         GCP          n2-standard-2     2       8         -              us-central1   0.10
+         IBM          bx2-8x32          8       32        -              us-east       0.38
+         Lambda       gpu_1x_a10        30      200       A10:1          us-east-1     0.60
+        ---------------------------------------------------------------------------------------------------.
+
+
+.. note::
+  SkyPilot will use the cluster and namespace set in the ``current-context`` in the
+  kubeconfig file. To manage your ``current-context``:
+
+  .. code-block:: console
+
+    $ # See current context
+    $ kubectl config current-context
+
+    $ # Switch current-context
+    $ kubectl config use-context mycontext
+
+    $ # Set a specific namespace to be used in the current-context
+    $ kubectl config set-context --current --namespace=mynamespace
+
+FAQs
+----
+
+* **Are autoscaling Kubernetes clusters supported?**
+
+  Yes - however they currently require adjusting the resource provisioning timeout (:code:`Kubernetes.TIMEOUT` in `clouds/kubernetes.py`) to a large value to give enough time for the cluster to autoscale. We are working on a better interface to adjust this timeout - stay tuned!
+
+* **What container image is used for tasks? Can I specify my own image?**
+
+  We use and maintain a SkyPilot container image that has conda and a few other basic tools installed. You can specify a custom image to use in `clouds/kubernetes.py`, but it must have rsync, conda and OpenSSH server installed. We are working on a interface to allow specifying custom images through the :code:`image_id` field in the task YAML - stay tuned!
+
+* **Can SkyPilot provision a Kubernetes cluster for me? Will SkyPilot add more nodes to my Kubernetes clusters?**
+
+  The goal of SkyPilot on Kubernetes is to run SkyPilot tasks on resources in an existing Kubernetes cluster. It does not provision any new Kubernetes clusters or add new nodes to an existing Kubernetes cluster. The Kubernetes control plane remains untouched.
+
+* **I have multiple users in my organization who share the same Kubernetes cluster. How do I provide isolation for their SkyPilot workloads?**
+
+  You can create an individual Kubernetes namespace for each user and set it in the kubeconfig distributed to the user. SkyPilot will use this namespace for running all tasks.
+
+Features and Roadmap
+--------------------
+
+SkyPilot on Kubernetes is under active development. Some features are in progress and will be released soon:
+
+* CPU Tasks - ✅ Available
+* Auto-down - ✅ Available
+* Storage mounting - ✅ Available on x86_64 clusters
+* GPU Tasks - ✅ Available
```

Comment:
move to top?

Reply:
I'd prefer to prioritize other information first - the benefits of using SkyPilot on k8s and how to get started. wdyt?

Reply:
I meant that L146 GPU Tasks should be moved to the top of the bullet list.